app/views/works/edit_multiple.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <%= render "works/work_abbreviated_list", :works => @works %>
 
-<%= form_for :work, :url => update_multiple_user_works_path(@user), :html => { :method => :put, :class => "post" } do |form| %>
+<%= form_for :work, :url => update_multiple_user_works_path(@user), method: :put, :html => { :class => "post" } do |form| %>
```

Comment:
It's a question David, I saw patch for the first time and read that it was the prefered way for updates however I don't know what is the right thing to do and this view is an edit.